metadata-ingestion/src/datahub/ingestion/source/looker/looker_usage.py

```diff
@@ -224,6 +224,11 @@ def get_id(self, looker_object: ModelForUsage) -> str:
     def get_id_from_row(self, row: dict) -> str:
         pass
 
+    @property
+    @abstractmethod
+    def SKIPPED_REPORT_KEY(self) -> str:
```

Comment:
what if this returned was "def report_skip_set(self) -> LossySet"

and the methods just were `return self.report.charts_skipped_for_usage` - would be cleaner and would allow mypy to check this stuff for us

Reply:
I like that better